mlflow/ml-package-versions.yml

```diff
@@ -66,63 +66,28 @@ tensorflow:
       pip install tf-nightly
 
   models:
-    minimum: "2.3.0"
+    minimum: "2.6.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       # Requirements to run tests for keras
-      "== dev": ["scikit-learn", "pyspark", "pyarrow"]
+      ">= 0.0.0": ["scikit-learn", "pyspark", "pyarrow"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py
-
-      # Run tests for keras against keras-nightly that's installed with tf-nightly:
-      # https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/tools/pip_package/setup.py#L110-L122
-      if [ "$PACKAGE_VERSION" == "dev" ]; then
-        pytest tests/keras/test_keras_model_export.py
-      fi
+      pip install keras==`python -c 'import tensorflow as tf; from packaging.version import Version; ver = Version(tf.__version__); print(f"{ver.major}.{ver.minor}")'`
```

Comment:
```suggestion
      pip install keras==$(pip freeze | grep 'tensorflow==' | grep -Eo '[0-9].[0-9]')
```

Reply:
There is still one failed test if we use keras version to be TFversion major.minor:
https://github.com/mlflow/mlflow/runs/8112708259?check_suite_focus=true#step:13:406

Reply:
I think we need keras version satisfy:
keras.major == tf.major and keras.minor == tf.minor and keras.micro <= tf.micro

Reply:
Seemingly it is caused by tensorflow-estimator version imcompatible.